doc/source/train/user_guide.rst

```diff
@@ -897,6 +897,53 @@ Checkpoints can be loaded into the training function in 2 steps:
 
 .. _train-fault-tolerance:
 
+Automatic Mixed Precision
+-------------------------
```

Comment:
Can you move this section to go after `Profiling`?

(Also, be careful of the `.. _train-fault-tolerance:` bookmark directly above!)

Reply:
+1

Reply:
Fixed. What's the motivation for moving the section to go after `Profiling`?

Also, what does the `.. _train-fault-tolerance:` bookmark do?